base/ltfssdcl.dtx

```diff
@@ -507,8 +569,138 @@
  \expandafter#1\ifx\math@bgroup\bgroup{#4}\else#4\fi
  }%
 }
+%    \end{macrocode}
+%
+%
+
+%  \begin{macro}{\freeze@math@version}
+% \changes{v3.0x}{2021/09/26}{Macro added for (gh/676)}
+%    This command stores the current state of the math version and sets
+%    things up to return to it after each formula from now on. We use
+%    L3 programming layer code to set it up.
+%    \begin{macrocode}
+\ExplSyntaxOn
+\cs_new:Npn\freeze@math@version #1 {
+%    \end{macrocode}
+%    Save the current \cs{mv@\meta{version}} code and the number of
+%    allocated mathgroups inside.
+%    \begin{macrocode}
+  \@font@info{Freeze~ math~ alphabet~ allocation~ in~ version~ #1}
+  \cs_gset_eq:cc { mv@#1@frozen }{ mv@#1 }
+  \tl_gset:cx { g__mv_frozen_ #1 _tl }{ \int_use:c { c@mv@#1 } }
```

Comment:
I think we should replace `__mv` by `__latexe`, or something like that, for “general” (not in a module of itself) `expl3` code in LaTeX2e, so we don't end up with hundreds of prefixes taken up by small pieces of code.

Reply:
I think you are right that "mv" is a bad module prefix, but in my opinion so is "latexe", just assume the whole kernel would be L3 based: if then each and every macro starts off with   `__latexe` that would be very unhelpful imo. Maybe the right prefix is "fss" (font selection scheme) or "nfss"  here, assuming that over time other parts of NFSS become L3 based.

Reply:
used nfss as this is a well known abeit not so "new" concept

Reply:
Yeah, `__latexe` would not be great either, but for "one offs" it would be less worse than registering a new prefix for just two or three macros. But `__nfss` is much better indeed, if we end up needing more of those in the future.